grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/tasks/internal/FormatBazelBuildFileTask.kt

```diff
@@ -48,7 +48,10 @@ abstract class FormatBazelFileTask : DefaultTask() {
     fun action() {
         if (bazelFile.exists()) {
             execOperations.exec {
-                commandLine = listOf("buildifier", bazelFile.absolutePath)
+                commandLine = listOf(
+                    "/tmp/buildifier",
```

Comment:
Instead of `/tmp` can we use project's build dir to keep these files scoped to project as well as platform independent? The `/tmp` might not sit well with windows.